skill/texts.py

```diff
@@ -0,0 +1,43 @@
+import random
+import pymorphy2
+
+morph = pymorphy2.MorphAnalyzer()
+
+
+def __inflect(word, case):
+    return " ".join([morph.parse(x)[-1].inflect(case).word for x in word.split(" ")])
+
+
+def hello():
+    text = "Привет! Это Цифровой дневник"
+    tts = "Привет! Это Цифровой дневник"
+    return text, tts
+
+
+def mistake():
+    text = """Прошу прощения. Во время игры возникла непредвиденная ошибка.
+    Я ее обязательно исправлю. Возвращайтесь чуть позже."""
+
+    return text, text
+
+
+def goodbye():
+    text = """Кажется игра у нас не клеится.
+    Ладно. Возвращайтесь в любое время.
+    До свидания!"""
```

Comment:
```suggestion
    text = """До свидания!
    Возвращайтесь в любое время."""
```

Тут тоже можно убрать "игру"